### PR TITLE
NOTES - HC ICAMA.vbs

### LINKED ISSUE #1816: New Script - HC ICAMA Notes script
Basic case noting script that details Adoption HC ICAMA.

Kelly will take this one.

### PR DESCRIPTION
**Pull request by khiestand**
BLIP: This is a new basic case noting script to note details of a HC ICAMA action.

Resolves Issue #1816

### CODE REVIEW

**Comment by theVKC** [comment]
You made the name of your script "create resolves #...". That isn't a valid name. Here's how to fix it:

1. Update the title of this pull request to remove the word "Create".
2. Update the name of the actual script to reflect the proper naming conventions for our scripts (it should be NOTES - HC ICAMA.vbs).

I'll now take a look at the script and give you additional comments in-line if needed. :)

File: Script Files/NOTES/Resolves #1816 NOTES - HC ICAMA.vbs [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+'Required for statistical purposes===========================================================================================
+STATS_counter = 1               'sets the stats counter at one
+STATS_manualtime = 300          'manual run time in seconds
+STATS_denomination = "C"        'C is for each case
+'END OF stats block==========================================================================================================
+ 
+'THE DIALOG----------------------------------------------------------------------------------------------------------
+
+BeginDialog HC_ICAMA, 0, 0, 286, 380, "HC ICAMA"
```
Comment (theVKC) [inline Script Files/NOTES/Resolves #1816 NOTES - HC ICAMA.vbs]: This dialog is called something different than below. It should be `HC_ICAMA_dialog`.

File: Script Files/NOTES/Resolves #1816 NOTES - HC ICAMA.vbs [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+EMConnect ""
+
+'Grabs the MAXIS case number            
+CALL MAXIS_case_number_finder(case_number)
+
+'Shows dialog
+DO
+	err_msg = ""		
+	Dialog HC_ICAMA_dialog
+		IF ButtonPressed = 0 THEN StopScript
```
Comment (theVKC) [inline Script Files/NOTES/Resolves #1816 NOTES - HC ICAMA.vbs]: This whole section is indented one too many. The IF statements should line up with the dialog. :)

File: Script Files/NOTES/Resolves #1816 NOTES - HC ICAMA.vbs [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+  DropListBox 120, 80, 100, 15, "Select One..."+chr(9)+"Adoption"+chr(9)+"Foster Care", type_dropdown
+  EditBox 105, 105, 115, 15, fc100
+  EditBox 105, 130, 115, 15, fcarep
+  EditBox 125, 155, 95, 15, ma_requested
+  EditBox 140, 180, 80, 15, aa_payment
+  EditBox 100, 205, 120, 15, ohc
+  EditBox 90, 230, 130, 15, pmap_ex
+  EditBox 125, 255, 95, 15, faxed_date
+  CheckBox 35, 280, 95, 15, "MA Coverage Form Sent", coverage_checkbox
+  CheckBox 35, 300, 245, 15, "Navigate to DAIL/WRIT to Create a TILK to Approve next 6 Month Budget", tikl_checkbox
```
Comment (theVKC) [inline Script Files/NOTES/Resolves #1816 NOTES - HC ICAMA.vbs]: It looks like you wrote "TILK" instead of "TIKL" here. :)

**Comment by theVKC** [comment]
OK, added my notes above @khiestand! Please email with any questions! :)

**Comment by khiestand** [comment]
@theVKC  I think/hope I got everything :-)

**Comment by theVKC** [comment]
Everything except one crucial piece, @khiestand... the title of the script itself is still broken.

image

Our script file titles can't look like this. Betting it was just a miscommunication about what needs to be marked with "Resolves #xxxx". Script file names should never be marked this way. Only pull request titles. :)

**Comment by khiestand** [comment]
Ok I removed it, yes I think I was maybe mis-reading the Pull Request instructions :-)

**Comment by theVKC** [comment]
Looks much better now!

**Pull request merged**